superset/models.py

```diff
@@ -2276,15 +2281,18 @@ def wrapper(*args, **kwargs):
                 params = json.dumps(d)
             except:
                 pass
+            value = f(*args, **kwargs)
             log = cls(
                 action=f.__name__,
                 json=params,
                 dashboard_id=d.get('dashboard_id') or None,
                 slice_id=slice_id,
+                duration_ms=(datetime.now() - start_dttm).total_seconds() * 1000,
+                referrer=request.referrer[:1000],
```

Comment:
It has the information of the previous url in the browser that got the person there. This is neat to track people coming from other domains.